src/app/home/partners/partners.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Observable } from 'rxjs';
+
+import { PartnerService } from 'src/app/core/services/partner.service';
+import { Partner } from 'src/app/core/model/partner';
+
+@Component({
+  selector: 'sw-partners',
+  templateUrl: './partners.component.html',
+  styleUrls: ['./partners.component.scss']
+})
+export class PartnersComponent implements OnInit {
+
+  partners = new Observable<Partner[]>();
```

Comment:
According to the notation for observables, this should be partners$. Add the $ symbol. Also, there's no need to initialize this, since this value is never read. You are already initializing it in the on init cycle.